File: Arnav Sharma/Assignment-1/count_negative.java
package com.array;

import java.util.Scanner;

public class count_negative {
        public static void main(String[] args) {
            int n;
            Scanner sc = new Scanner(System.in);
            System.out.println("enter the no of elements");
            n = sc.nextInt();

            int[] arr = new  int[15];
            System.out.println("Enter the elements");
            for (int i=0; i<n; i++){
                arr[i] = sc.nextInt();
            }
            {
                int count=0;
                for(int i=0;i<n;i++){
                    if(arr[i]<0)
                        count++;
                }
                System.out.println("no of negative elements are: "+count);
            }
        }
}

File: KM POOJA/Assignment 1/countNegative.cpp

//time complexity O(n) and space O(1)
#include <iostream>
using namespace std;
int main()
{
    int i, n;
    /* Input array size */
    cout<<"Enter size of array: ";
     cin>>n;
     int arr[n];
    /* Input elements in array */
    cout<<"Enter "<<n<<" elements in the array : ";
    for(i=0; i<n; i++)
    {
      cin>>arr[i];
    }
    int countNegative=0;
    
    for(i=0; i<n; i++)
    {   if(arr[i]<0)
         countNegative++;
    }
    cout<<"There are total "<<countNegative<<" negative elements present in array..."<<endl;
    return 0;
}